client/state/ui/happychat/reducer.js

```diff
@@ -0,0 +1,15 @@
+import { combineReducers } from 'redux';
+
+import {
+	HAPPYCHAT_OPEN,
+} from 'state/action-types';
+
+const open = ( state = false, action ) => {
+	switch ( action.type ) {
+		case HAPPYCHAT_OPEN:
+			return action.isOpen !== undefined ? action.isOpen : state;
```

Comment:
Is there a case where `isOpen` wouldn't be defined? We could just return as `!! action.isOpen`.